scripts/plotting/polar_map.py

```diff
@@ -256,7 +255,7 @@ def polar_map(adfobj):
                                         hemi = "SH"
                                     #End if
                                     
-                                    fig = make_polar_plot(mseasons[s], oseasons[s], dseasons[s], hemisphere=hemi, **vres)
+                                    fig = pf.make_polar_plot(plot_name, mseasons[s], oseasons[s], dseasons[s], hemisphere=hemi, **vres)
```

Comment:
If you remove the `return fig` statement from the `make_polar_plot` function then this can be changed to:

```suggestion
                                    pf.make_polar_plot(plot_name, mseasons[s], oseasons[s], dseasons[s], hemisphere=hemi, **vres)
```

Also the `fig.savefig` and `plt.close` calls below should be deleted.